### ISSUE TITLE
Text summary: Error encountered when serializing __tensorboard_plugin_asset__tensorboard_text.

### ISSUE THREAD

**Issue by cgorman** [tensorflow/tensorflow#10024]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux CentOS 7
- **TensorFlow installed from (source or binary)**: Source
- **TensorFlow version (use command below)**: `v1.1.0-rc2-1185-ge4f5123' 1.2.0-rc0`
- **Bazel version (if compiling from source)**: `0.4.5`
- **CUDA/cuDNN version**: `8.0` / `5.1.10`
- **GPU model and memory**: GeForce GTX Titan X 12GB
- **Exact command to reproduce**:

```
import tensorflow as tf

logdir = "/output_dir"
text = "A text summary"
text_tensor = tf.constant(text)
text_summary = tf.summary.text("Summary", text_tensor)
merged = tf.summary.merge([text_summary])
writer = tf.summary.FileWriter(logdir, tf.get_default_graph())
s = tf.Session()
s.run(tf.global_variables_initializer())
summary = s.run(merged)
writer.add_summary(summary)
```

### Describe the problem
Tensorflow  gives the following error when I try to create a summary writer when the graph has a text summary:
```
WARNING:tensorflow:Error encountered when serializing __tensorboard_plugin_asset__tensorboard_text.
Type is unsupported, or the types of the items don't match field type in CollectionDef.
'TextSummaryPluginAsset' object has no attribute 'name'
```
On my larger project I was able to output some text to the summary page on tensorboard, but the Markdown I put in there wasn't being parsed at all. I don't know if these problems are related but I feel like they probably are.

### Source code / logs
Code in first section is an MWE

**Comment by skye** [comment]
@dandelionmane @jart

**Comment by Netzeband** [comment]
I have the same issue. It appears, when I store a checkpoint of my model.

OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
TensorFlow version (use command below): tensorflow_gpu-1.2.0-cp35-cp35m-win_amd64.whl  from PIP

**Comment by dandelionmane** [comment]
Ah, this is caused by an interaction between the way text summary was implemented and the metagraphdef. The warning doesn't actually break anything, it's just a warning. 

This is unrelated to any markdown parsing issues.

@chihuahua just submitted a PR internally that fixes this by using a different mechanism for the text summary. The fix will be in tf 1.3. Closing this since the issue is fixed at head.
